apis/fluent/atrium-api-fluent/src/commonMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/rangeExpectations.kt

```diff
@@ -0,0 +1,41 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.logic.isInRange
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.logic._logicAppend
+
+/**
+ * Expects that the subject of `this` expectation is within the [expected] range (inclusive).
```

Comment:
detail, you can also keep as is. Personally, I would remove the `(inclusive)` although it is correct
```suggestion
 * Expects that the subject of `this` expectation is within the [expected] range.
```
because this could confuse someone who writes
 ```
expect('d').toBeInRange('a' until 'c')
```
Where `c` is exclusive (behind the seen it is still a ClosedRange but endInclusive is `b`.) This comes apparent in error reporting, the user will see `a .. b` instead of `a ..< c` (I wonder if kotlin will introduce an own type at some point, something like IntOpenEndRange).